src/main/java/Inventory.java

```diff
@@ -0,0 +1,55 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class Inventory {
+
+    private ArrayList<Item> items;
+
+    public Inventory(ArrayList<Item> items) {
+        this.items = items;
+    }
+
+    public ArrayList<Item> getInventory() { return items; }
+
+    public void addToInventory(Item item) { this.items.add(item); }
+
+    public String getStringInventory() {
+        String playerInventory = "";
+
+        for (Item item : items){
+            playerInventory = item.getNameFromItem() + ", " + playerInventory;
+        }
+
+        int unnecessaryComma = playerInventory.lastIndexOf(",");
+
+        return playerInventory.substring(0, unnecessaryComma);
+    }
+
+    public static String printInventory(ArrayList<Item> playerItems) {
+        String playerInventory = "";
+        String playerInventoryFinal;
+
+        for (Item item : playerItems){
+            playerInventory = item.getNameFromItem() + ", " + playerInventory;
+        }
+        int unnecessaryComma = playerInventory.lastIndexOf(",");
+        playerInventoryFinal = "Your current inventory: " + playerInventory.substring(0, unnecessaryComma);
+
+        return playerInventoryFinal;
```

Comment:
Seems a little redundant. There may be a better way to do this while only using one method instead of two.